internal/run/templates/templates.go

```diff
@@ -29,10 +30,11 @@ type Templates struct {
 func Parse(renderTermColors RenderTermColorsType) *Templates {
 	templateFunctions := template.FuncMap{
 		"rate": func(duration time.Duration, count uint64) uint64 {
-			if uint64(duration/time.Second) == 0 {
+			if count == 0 || duration == 0 {
 				return 0
 			}
-			return count / uint64(duration/time.Second)
+
+			return uint64(math.Round(float64(count) / duration.Round(time.Second).Seconds()))
```

Comment:
I think you might end up with a divide by zero error here if `duration < 0.5`. Rounding could be done above the == 0 zero guard to protect against that.